app/views/layouts/_navbar.html.erb

```diff
@@ -7,10 +7,12 @@
     <div class="collapse navbar-collapse" id="navbarSupportedContent">
       <ul class="navbar-nav me-auto mb-2 mb-lg-0">
       </ul>
-      <form class="d-flex" role="search">
-        <input class="form-control me-2" type="search" placeholder="Search" aria-label="Search">
+      <form class="d-flex" role="search" action="search/index">
```

Comment:
The action here should be `/search/index` (or do `get 'search/index' as: :search` in `routes.rb` and then you can do `search_path` here) otherwise you end up trying to go places that don't exist depending on what page you're on.